android/filament-android/src/main/java/com/google/android/filament/Renderer.java

```diff
@@ -45,6 +45,20 @@
     private final Engine mEngine;
     private long mNativeObject;
 
+    /**
+     * Information about the display this renderer is associated to
+     */
+    public static class DisplayInfo {
+        /** refresh-rate of the display in Hz. set to 0 for offscreen or turn off frame-pacing. */
+        public float refreshRate = 60.0f;
```

Comment:
Do you set it to 30 to do 30fps rendering? We also have a refresh rate for dynamic resolution, should we have both?

Reply:
No, that is the refresh of the display, later I will add the concept of target framerate.
The other setting for dynamic resolution will probably go away eventually.

I'm first adding the new stuff, and I'll remove the old stuff as a second step.